docs/src/notebooks/9_max_min_driving_force_analysis.jl

```diff
@@ -0,0 +1,198 @@
+
+# # maximum minimum driving force analysis
+
+#md # [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/notebooks/@__NAME__.ipynb)
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/notebooks/@__NAME__.ipynb)
+
+# In this notebook we will use max-min driving force analysis (MMDFA) to find optimal concentrations
+# for the metabolites in glycolysis to ensure that the smallest driving force across all the
+# reactions in the model is as large as possible. For more information, see Flamholz, Avi, et al.
+# "Glycolytic strategy as a tradeoff between energy yield and protein cost.", Proceedings of
+# the National Academy of Sciences 110.24 (2013): 10039-10044.
+
+using COBREXA
+using Tulip
+
+# Create a model of glycolysis fermentation.
+
+#md # !!! tip "Directions of reactions"
+#md #     Be careful when constructing models for MMDFA, the reaction directions in the model
+#md #     and thermodynamic data need to be consistent with the overall flux direction implied by the model.
+#md #     For example, in BiGG, `LDH_D` is written
+#md #     `lac__D + nad ⟷ h + nadh + pyr` and the associated ΔrG'⁰ is 23.6803 kJ/mol.
+#md #     For MMDFA no flux is calculated, so you need to write the reaction
+#md #     in the direction of flux, i.e. `h + nadh + pyr ⟶ lac__D + nad` with ΔrG'⁰ as
+#md #     -23.6803 kJ/mol.
+
+model = StandardModel("Glycolysis")
+
+mets = [
```

Comment:
btw where did you get the reactions? did you write them by hand? (maybe better publish the model then) Or are they from some other model? (maybe better take that model and kill all reactions you're not interested in)

Reply:
This model is from the paper that introduced the method :) And yep, I wrote them by hand. Since you do not calculate fluxes you need to be careful about the direction you write the reaction in, usually metabolic models calculate fluxes so they do not care about how the reaction is written. This is why the note at the top of the notebook exists

Reply:
It would be more of a hassle to strip it out of a model than to just do it by hand... Anyway, these reactions are so well known, everybody will know what they are/where they are from

Reply:
> everybody

what about me? :D